languages/tree-sitter-stack-graphs-java/src/stack-graphs.tsg

```diff
@@ -185,28 +359,95 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
   edge member -> def
 }
 
+(method_declaration (formal_parameters (_) @param)) @method
```

Comment:
I've tried to avoid making nodes for the (many) different kinds of intermediate nodes that the Java grammar defines (as I recall, it mirrors the Java standard's grammar fairly directly). That results in slightly leaner paths, but more importantly (IMO), somewhat less cluttered stack graph rules.